packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -292,7 +300,8 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         // Price has never been requested.
         uint32 currentRoundId = getCurrentRoundId();
         if (_getRequestStatus(priceRequest, currentRoundId) == RequestStatus.NotRequested) {
-            uint32 roundIdToVoteOn = currentRoundId + 1; // Vote on request in the following round.
+            uint32 roundIdToVoteOn = getRoundIdToVoteOnRequest(currentRoundId + 1);
```

Comment:
Thoughts on just using the length of the pendingRequests array to determine how many rounds out we need to go instead of tracking?